Softeq.XToolkit.Common.Tests/WeakTests/WeakDelegatesTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+#nullable disable
```

Comment:
Can be removed, because disabled on the top level (for all Tests projects)

Reply:
Fixed